sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -610,17 +611,12 @@ export class CryptographyClient {
   /**
    * The base URL to the vault
    */
-  public readonly vaultUrl: string;
+  private readonly vaultUrl: string;
 
   /**
    * The options to create the connection to the service
    */
-  public readonly pipeline: Pipeline;
-
-  /**
-   * The authentication credentials
-   */
-  protected readonly credential: ServiceClientCredentials | TokenCredential;
+  private readonly pipeline: Pipeline;
```

Comment:
This need not be a private member either. Pipeline is used only in the constructor